examples/js/scatterplot.js

```diff
@@ -754,4 +754,38 @@ function Scatterplot(el, component, svgClass) {
     return str.replace(' ', '').toLowerCase()
   }
 }
+
+  var scatterplot1Tab1 = document.querySelector('.scatterplot-1-tab-1');
```

Comment:
Only thing I see lacking for it to be accessible is when:
`.two:hover ~ hr {`

should also include `focus` along with `hover`.